pkg/controllers/xrd/reconciler.go

```diff
@@ -41,9 +42,9 @@ import (
 	rresource "github.com/crossplane/crossplane-runtime/pkg/resource"
 	"github.com/crossplane/crossplane/apis/apiextensions/v1alpha1"
 	"github.com/crossplane/crossplane/apis/apiextensions/v1alpha1/ccrd"
-	crequirement "github.com/crossplane/crossplane/pkg/controller/apiextensions/requirement"
+	cclaim "github.com/crossplane/crossplane/pkg/controller/apiextensions/claim"
```

Comment:
Nit: These `cclaim` and `rresource` names threw me off when reviewing the original agent PR, because they look a lot like typos.

Reply:
Changed with `coreclaim`